src/ingest_validation_tools/table-schemas/antibodies.yaml

```diff
@@ -0,0 +1,33 @@
+fields:
+  # NOTE: Example had "dataset_uuid": Won't work. Not known at submission time.
```

Comment:
How we could associate antibody metadata on each row of an antibody.tsv to the relevant dataset before a uuid is generated?

Reply:
>> I was thinking this would be referenced from the dataset metadata.tsv... Does that match your understanding?

> Yes. This is the suggested mechanism for now: Users should create a directory called "extras" and put this Ab tsv and the Ab validation report in it.

From the quotation above, I was thinking that each dataset would reference an entire antibody list... Is it clear to you how that is different from having the antibodies rows reference datasets?

Let me finish integrating the other changes here, and then let's make time to talk this through. Probably hard to resolve messaging back and forth.